syntax = "proto3";

import "simpletest.proto";

package qtprotobufnamespace.tests;

service TestService {
  rpc testMethod (SimpleIntMessage) returns (SimpleIntMessage) {}
} 
